cmd/shared.go

```diff
@@ -142,7 +148,8 @@ func ProcessMR(ctx context.Context, client scm.Client, cfg *config.Config, event
 		return fmt.Errorf("Configuration failed validation: %w", err)
 	}
 
-	// Write the config to context so we can pull it out later
+	// Write the config to context so we can pull it out later, if a global config file was set
+	// this overrides the global config with the merge global and repository config
```

Comment:
Maybe it's because I haven't finished my coffee yet, but I'm not sure how to interpret this comment.  Would either of these alternatives be more precise?

```suggestion
	// Write the config to context so we can pull it out later, but only if a global config file was set
	// This overrides the global config with the merged global and repository config
```

or

```suggestion
	// Write the config to context so we can pull it out later
	// If a global config file was set, this overrides the global config with the merged global and repository config
```

(I _think_ the second one is what we want? Maybe?)

Reply:
yup the second one is what we'd want